benchmarks/spawn.py

```diff
@@ -42,17 +49,42 @@ def run_spawn_n_kw():
     eventlet.spawn_n(dummy, i=1)
 
 
+def linked(gt):
+    pass
+
+
+def run_spawn_link1_wait():
+    t = eventlet.spawn(dummy)
+    t.link(linked)
+    t.wait()
+
+
+def run_spawn_link5_wait():
+    t = eventlet.spawn(dummy)
+    t.link(linked)
+    t.link(linked)
+    t.link(linked)
+    t.link(linked)
+    t.link(linked)
+    t.wait()
+
+
 best = benchmarks.measure_best(
     5, iters,
     'pass',
     cleanup,
     run_spawn_n,
     run_spawn,
-    run_spawn_n_kw)
-print("eventlet.spawn", best[run_spawn])
-print("eventlet.spawn_n", best[run_spawn_n])
-print("eventlet.spawn_n(**kw)", best[run_spawn_n_kw])
-print("%% %0.1f" % ((best[run_spawn] - best[run_spawn_n]) / best[run_spawn_n] * 100))
+    run_spawn_n_kw,
+    run_spawn_link1_wait,
+    run_spawn_link5_wait,
+)
+print(format_result("eventlet.spawn", best[run_spawn], iters))
+print(format_result("eventlet.spawn_n", best[run_spawn_n], iters))
+print(format_result("eventlet.spawn_n(**kw)", best[run_spawn_n_kw], iters))
+print(format_result("eventlet.spawn.link[1]+wait", best[run_spawn_link1_wait], iters))
+print(format_result("eventlet.spawn.link[5]+wait", best[run_spawn_link5_wait], iters))
+print("spawn/spawn_n: {:0.1f} %".format((best[run_spawn] - best[run_spawn_n]) / best[run_spawn_n] * 100))
```

Comment:
Maybe comparison of results should itself become a helper function, for lines 87 and 118?

Reply:
That would help address the pep8 style issues.

Reply:
Fixed benchmarks in a different way.